### PR TITLE
[MOSIP-29616]Updated Charts mosip-file-server 1.2.0.1

### PR DESCRIPTION
**Pull request by abhishek8shankar**

### CODE REVIEW

File: charts/mosip-file-server/values.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ service:
 image:
   registry: docker.io
   repository: mosipqa/mosip-file-server
-  tag: 1.2.0.1
+  tag: develop
```
Comment (ckm007) [inline charts/mosip-file-server/values.yaml:55]: It should be release-1.2.0.1 na?

Comment (abhishek8shankar) [reply charts/mosip-file-server/values.yaml:55]: release-1.2.0.1 branch is not present , so syed suggested to update in 1.2.0.1 .

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: charts/mosip-file-server/templates/virtualservice.yaml [hunk 2]
```diff
@@ -13,32 +13,31 @@ metadata:
   {{- end }}
 spec:
   hosts:
-  - "*"
+    - "*"
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/virtualservice.yaml]: reverted

File: charts/mosip-file-server/templates/servicemonitor.yaml [hunk 3]
```diff
@@ -33,4 +33,4 @@ spec:
       - {{ .Release.Namespace | quote }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
-{{- end }}
+  {{- end }}
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/servicemonitor.yaml]: removed space .

File: charts/mosip-file-server/templates/secrets.yaml [hunk 4]
```diff
@@ -14,7 +13,7 @@ metadata:
...
 data:
   {{- if .Values.mosipfileserver.secrets }}
-  {{- range $key, $value := .Values.mosipfileserver.secrets }}
-  {{ $key }}: {{ $value | b64enc | quote }}
-  {{- end }}
-  {{- end }}
+    {{- range $key, $value := .Values.mosipfileserver.secrets }}
+    {{ $key }}: {{ $value | b64enc | quote }}
+    {{- end }}
+    {{- end }}
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/secrets.yaml]: removed space

File: charts/mosip-file-server/templates/gateway.yaml [hunk 5]
```diff
@@ -8,11 +8,11 @@ spec:
...
-{{- end }}
-{{- end }}
+    - port:
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
+        - {{ .Values.mosipfileserver.host }}
+  {{- end }}
+  {{- end }}
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/gateway.yaml]: removed space

File: charts/mosip-file-server/templates/extra-list.yaml [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {{- range .Values.extraDeploy }}
 ---
 {{ include "common.tplvalues.render" (dict "value" . "context" $) }}
-{{- end }}
+  {{- end }}
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/extra-list.yaml]: removed space

File: charts/mosip-file-server/templates/deployment.yaml [hunk 7]
```diff
@@ -101,29 +101,29 @@ spec:
...
+              - configMapRef:
+                  name: {{ . }}
+              {{- end }}
+              {{- end }}
+              {{- if .Values.extraEnvVarsSecret }}
+              {{- range .Values.extraEnvVarsSecret }}
+              - secretRef:
+                  name: {{ . }}
+              {{- end }}
+              {{- end }}
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply charts/mosip-file-server/templates/deployment.yaml]: removed space

**Pull request merged**